lib/stream.js

```diff
@@ -34,13 +34,17 @@ const compose = require('internal/streams/compose');
 const { destroyer } = require('internal/streams/destroy');
 const eos = require('internal/streams/end-of-stream');
 const internalBuffer = require('internal/buffer');
+const duplexify = require('internal/streams/duplexify');
 
 const promises = require('stream/promises');
 
 const Stream = module.exports = require('internal/streams/legacy').Stream;
 Stream.Readable = require('internal/streams/readable');
 Stream.Writable = require('internal/streams/writable');
 Stream.Duplex = require('internal/streams/duplex');
+Stream.Duplex.from = function from(body) {
+  return duplexify(body, 'body');
+};
```

Comment:
Shouldn't this be defined in `internal/streams/duplex` instead?

Reply:
That leads to recursion.